test/parallel/test-http-pipeline-serverresponse-assertionerror.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const common = require('../common');
+
+// This test ensures that Node.js doesn't crash with an AssertionError at
+// `ServerResponse.resOnFinish`
```

Comment:
More helpful would be a comment that says what made this case special. (E.g., "This test ensures that Node.js deals with a large number of HTTP requests correctly." or some such.) Ditto with the test file name.

Reply:
Will do, thanks.